Marsiple/Common/Network/HTTPRequestHeader.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  HTTPRequestHeaders.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 18/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import Foundation
+
+struct HTTPRequestHeader {
```

Comment:
Hmm, what do you think about using a `enum` instead of a `struct` here? 

The primary benefit of using an `enum` is that we get a compiler-checked, finite list of options. The call-site will look almost identical, with an addition of `rawValue` (if we used `String` as the `enum` type).

Not mandatory, this is open for debate.